src/v/model/record.h

```diff
@@ -470,6 +470,14 @@ struct producer_identity
     auto serde_fields() { return std::tie(id, epoch); }
 };
 
+struct tx_range {
```

Comment:
tx_range is used as a part of the tx snapshot (it is being serialized to disk) so it make sense to add a comment to prevent people from changing this struct (without supporting it in the serialization area)

Reply:
Done